src/main/resources/application-fga.yml

```diff
@@ -0,0 +1,51 @@
+server:
```

Comment:
This file is not used by jUnit tests, probably it has been added by mistake

Reply:
I was using it for manual tests by adding the proper "fga" profile. For me it is useful, but we can remove it if you prefer